server_development/topics/auth-spi.adoc

```diff
@@ -658,8 +658,7 @@ The config values can be defined within the admin console if you set up config d
     }
 ----
 
-Last method in SecretQuestionCredentialProvider class is getCredentialTypeMetadata(), which is an abstract method of CredentialProvider
-interface. Each Credential provider has to provide and implement this method. The method returns an instance of CredentialTypeMetadata,
+Last method in SecretQuestionCredentialProvider class is getCredentialTypeMetadata(CredentialTypeMetadataContext metadataContext), which is an abstract method of CredentialProvider interface. Each Credential provider has to provide and implement this method. The method returns an instance of CredentialTypeMetadata,
```

Comment:
Please add "the" before "Last method" and "CredentialProvider interface."

Reply:
Done